src/main/java/racingCar/service/RacingCarsService.java

```diff
@@ -0,0 +1,42 @@
+package racingCar.service;
+
+import static racingCar.view.Output.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import racingCar.model.RacingCars;
+import racingCar.utlis.Convertor;
+import racingCar.utlis.Util;
+import racingCar.validator.CountValidator;
+import racingCar.view.Output;
+
+public class RacingCarsService {
+
+	private RacingCars cars;
+	private int count;
+
+	public void initiateCars(String nameString) throws Exception {
+		cars = new RacingCars(new ArrayList<>(Arrays.asList(
+			Convertor.separate(nameString)
```

Comment:
```suggestion
               String[] names = Convertor.separate(nameString);
               cars = new RacingCars(Arrays.asList(names));
```
한번에 많은 것들을 하기보다 하나씩 단계별로 진행을 하는 것이 명확해요.

Reply:
오 정말 보기 훨씬 명확하네요!😆 
메소드가 너무 중첩되어 복잡해질 땐 중간에 변수로 한번씩 끊어주도록 할게요!